test/e2e_node/builder/build.go

```diff
@@ -139,7 +140,16 @@ func getK8sBin(bin string) (string, error) {
 
 // GetKubeletServerBin returns the path of kubelet binary.
 func GetKubeletServerBin() string {
-	bin, err := getK8sBin("kubelet")
+	var kubeletServer string
```

Comment:
Is it a good assumption that we'll always run from the same platform? (EG what if I want to build and run e2e-node on linux but target a windows remote node?)

We also have https://github.com/kubernetes/kubernetes/blob/fb25b8dd7ba054ad428d37eb2d3aec4d68d43c45/test/e2e_node/builder/build.go#L34